package.json

```diff
@@ -21,12 +21,13 @@
     "gulp-connect": "^2.2.0",
     "gulp-if": "^2.0.0",
     "gulp-inject": "^3.0.0",
-    "gulp-less": "^3.0.5",
     "gulp-load-plugins": "^1.1.0",
     "gulp-preprocess": "^2.0.0",
     "gulp-protractor": "^2.1.0",
     "gulp-recess": "^1.2.0",
```

Comment:
I didn't noticed but in red with minus in front of the line means i deleted it !
That's why i don't have it anymore.